src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookConsumer.java

```diff
@@ -98,7 +73,11 @@ private void triggerJob(RefsChangedWebhookEvent event,
 
     private static Collection<? extends SCM> getScms(ParameterizedJobMixIn.ParameterizedJob<?, ?> job) {
         SCMTriggerItem triggerItem = SCMTriggerItem.SCMTriggerItems.asSCMTriggerItem(job);
-        if (triggerItem != null) {
+        if (triggerItem instanceof WorkflowJob) {
+            if (((WorkflowJob) triggerItem).getDefinition() instanceof CpsScmFlowDefinition) {
+                return Collections.singletonList(((CpsScmFlowDefinition) ((WorkflowJob) triggerItem).getDefinition()).getScm());
```

Comment:
Is workflow-cps a core plugin? this change will add a new dependency to our plugin.

Reply:
yes, part of pipeline, this does not change our dependency requirements.